src/runtime_src/xocl/CMakeLists.txt

```
@@ -106,12 +106,18 @@ install(TARGETS xilinxopencl
   RUNTIME DESTINATION ${XRT_INSTALL_BIN_DIR}
 )
 
-install(TARGETS xilinxopencl xilinxopencl_static
+install(TARGETS xilinxopencl
   EXPORT xrt-dev-targets
   ARCHIVE DESTINATION ${XRT_INSTALL_LIB_DIR} COMPONENT ${XRT_DEV_COMPONENT}
   LIBRARY DESTINATION ${XRT_INSTALL_LIB_DIR} COMPONENT ${XRT_DEV_COMPONENT} ${XRT_NAMELINK_ONLY}
 )
 
+install(TARGETS xilinxopencl xilinxopencl_static
```

```suggestion
install(TARGETS xilinxopencl_static
```
oops.  this was a typo.